src/applications/financial-status-report/components/InstallmentContract.jsx

```diff
@@ -0,0 +1,310 @@
+import React, { useState } from 'react';
+import { connect } from 'react-redux';
+import { setData } from 'platform/forms-system/src/js/actions';
+import {
+  VaTextInput,
+  VaDate,
+} from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+import { parseISODate } from 'platform/forms-system/src/js/helpers';
+import { isValidCurrency } from '../utils/validations';
+
+const defaultRecord = [
+  {
+    contractType: '',
+    creditorName: '',
+    originalLoanAmount: '',
+    unpaidBalance: '',
+    amountDueMonthly: '',
+    from: '',
+    amountOverdue: '',
+  },
+];
+
+const InstallmentContract = props => {
+  const { data, goToPath, setFormData } = props;
+
+  const { expenses } = data;
+  const { installmentContracts = [] } = expenses;
+
+  const searchIndex = new URLSearchParams(window.location.search);
+  let editIndex = parseInt(searchIndex.get('index'), 10);
+  if (Number.isNaN(editIndex)) {
+    editIndex = installmentContracts?.length ?? 0;
+  }
+  const isEditing = editIndex >= 0 && !Number.isNaN(editIndex);
```

Comment:
Might consider changing to `const isEditing = !Number.isNaN(editIndex) && editIndex >= 0;`
This why you're checking to see if editIndex is a valid number first before comparing it with zero